security/registries.adoc

```diff
@@ -0,0 +1,104 @@
+[[security-registries]]
+= Registries
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-registries-where-do-your-containers-come-from]]
+== Where Do Your Containers Come From?
+
+As discussed, there are tools you can use to scan and track the contents of your
+downloaded and deployed container images. However, there are many public sources
+of container images. When using public container registries, you can add a layer
+of protection by using trusted sources.
+
+[[security-registries-immutable-and-certified-containers]]
+== Immutable and Certified Containers
+
+Consuming security updates is particularly important when managing _immutable
+containers_. Immutable containers are containers that will never be changed
+while running.
+
+Container images are a packaged, layered collection of binaries. When a bug is
+discovered in a deployed binary, you do not edit the binary in place; you fix
+the bug in the source code, rebuild the binary, and replace the broken binary
+with the newly rebuilt binary.
+
+When you deploy immutable containers, you do not step into the running container
+to replace one or more binaries; you rebuild and redeploy an updated container
+image.
+
+The list of known vulnerabilities is constantly evolving, so you need to track
+the contents of your deployed container images, as well as newly downloaded
+images, over time. Red Hat Security Advisories (RHSAs) will alert you to any
+newly discovered issues in Red Hat certified container images and direct you to
+the updated image.
+
+Certification of a container from Red Hat means that:
+
+- The container image is free of known vulnerabilities in the platform components
+or layers.
+- The container is compatible across the RHEL platforms, from bare metal to cloud.
+- The complete stack is supported by Red Hat and partners.
+
+[discrete]
+[[security-registries-further-reading-1]]
+==== Further Reading
+
+- More on immutable containers in {product-title}:
+** _{product-title} Architecture_: xref:{sourcedir}architecture/core_concepts/builds_and_image_streams.adoc#image-streams[Image Streams]
+** _{product-title} Developer Guide_: xref:{sourcedir}dev_guide/managing_images.adoc#referencing-images-in-image-streams[Referencing Images in Image Streams]
```

Comment:
We may want to link to https://access.redhat.com/blogs/product-security/posts/container-security-scoring